chat-server/src/routes/conversations/conversationsRouter.ts

```diff
@@ -48,6 +54,37 @@ export interface ConversationsRateLimitConfig {
   addMessageSlowDownConfig?: Partial<SlowDownOptions>;
 }
 
+export type AddCustomDataFunc = (
```

Comment:
Should we have doc comment for this type too?

Reply:
seems reasonable. will add